server/initializer/initializer.go

```diff
@@ -0,0 +1,91 @@
+package initializer
+
+import (
+	"github.com/MacroPower/macropower-analytics-panel/server/cacher"
+	"github.com/MacroPower/macropower-analytics-panel/server/payload"
+	"github.com/MacroPower/macropower-analytics-panel/server/worker"
+	"github.com/go-kit/kit/log"
+	"github.com/google/uuid"
+	"time"
+)
+
+func InitializeMetricsForDashboards(api worker.Client, logger log.Logger, cache *cacher.Cacher) {
```

Comment:
misleading name, as it doesn't initialize metrics, but instead adds a panel that starts sending metrics?

Reply:
It emulates visits to a dashboards to init metrics with 0 values. Username is used to compare if its emulated visit or not. If its emulated then session count (metric value) is not increased.